sdks/python/apache_beam/io/gcp/big_query_query_to_table_it_test.py

```diff
@@ -148,6 +148,7 @@ def test_big_query_legacy_sql(self):
                   'output': self.output_table,
                   'output_schema': DIALECT_OUTPUT_SCHEMA,
                   'use_standard_sql': False,
+                  'native': True,
```

Comment:
Maybe we should add a `--native` parameter? This would make switching between difference sources/sinks much easier.

Reply:
a native parameter for BQ Sink? (_ReadFromBQ(native=False/True) - > expand depending on the parameter?)

Reply:
I though about something else (a native parameter for this test), but now I see it's already here: https://github.com/apache/beam/blob/2bd3c5c0860846d47218b2d8715dc84e2161bdcc/sdks/python/apache_beam/io/gcp/big_query_query_to_table_pipeline.py#L55
so it's fine.

Reply:
But your idea sounds good. I think we could even remove the underscore, set `True` as the default value of `native` and expose that transform to users (not in this PR though) WDYT?